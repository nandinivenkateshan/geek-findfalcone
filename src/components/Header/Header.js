import React from 'react'
import styled from 'styled-components'

const Heading = styled.h1`
text-align:center;
`
const Aside = styled.aside`
display:flex;
justify-content: flex-end;

`
const StyledLink = styled.button`
font-size:19px;
padding:20px;
border: none;
background:none;
cursor:pointer;
color: #3b403c;
`
const GeekLink = styled.a`
font-size:20px;
padding:20px;
text-decoration: none;
color:#3b403c;
`

function Header () {
  const handleReset = () => {
    window.location.reload()
  }

  return (
    <header>
      <Aside>
        <StyledLink to='' onClick={handleReset}>Reset</StyledLink>
        <GeekLink href='https://www.geektrust.in/' target='new'>Geektrust Home</GeekLink>
      </Aside>
      <Heading> Finding Falcone !</Heading>
    </header>
  )
}

export default Header
