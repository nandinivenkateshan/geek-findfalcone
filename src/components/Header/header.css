
.findFalcone__header {
    text-align: center;
    margin: 70px;
}

