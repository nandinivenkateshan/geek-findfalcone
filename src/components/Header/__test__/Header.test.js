import React from 'react'
import { render, cleanup } from '@testing-library/react'
import Header from '../Header'

afterEach(cleanup)

it('renders', () => {
  const { asFragment } = render(<Header />)
  expect(asFragment()).toMatchSnapshot()
})
