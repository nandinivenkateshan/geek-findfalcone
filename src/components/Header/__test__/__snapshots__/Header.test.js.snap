// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <header
    class="sc-AxjAm gWmjHI"
  >
    <aside
      class="sc-AxiKw dDwdV"
    >
      <button
        class="sc-AxhCb cMlMFk"
        to=""
      >
        Reset
      </button>
      <a
        class="sc-AxhUy mdJCr"
        href="https://www.geektrust.in/"
        target="new"
      >
        Geektrust Home
      </a>
    </aside>
    <h1
      class="sc-AxirZ jmdGFA"
    >
       Finding Falcone !
    </h1>
  </header>
</DocumentFragment>
`;
